core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -737,7 +741,8 @@ bool core_mmu_entry_to_finer_grained(struct core_mmu_table_info *tbl_info,
 #endif
 	assert(prtn);
 
-	if (tbl_info->level >= 3 || idx > tbl_info->num_entries)
+	if (tbl_info->level >= XLAT_TABLE_LEVEL_MAX ||
+	    idx > tbl_info->num_entries)
```

Comment:
by the way, i wonder if this is bugged. Shouldn't it  be `idx >= tbl_info->num_entries)`?

Reply:
agree. maybe this bug is the origin for a similar bug introduced into the _copy() function above :)